packages/cuba-react-core/src/data/Instance.tsx

```diff
@@ -62,15 +64,31 @@ export class DataInstanceStore<T> implements DataContainer {
 
   @action
   update(entityPatch: Partial<T>): Promise<any> {
+    const normalizedPatch: Record<string, any> = this.normalizePatch(entityPatch, this.entityName);
+    Object.assign(this.item, normalizedPatch);
+    return this.commit();
+  }
+
+  normalizePatch(entityPatch: Record<string, any>, entityName: string): Record<string, any> {
```

Comment:
Need jsdocs here. Seems that I don't understand what happend in method. May be test also can help to understand.

Reply:
I'll do it as part of my upcoming PR for one-to-many compositions